attributes/default.rb

```diff
@@ -62,10 +62,14 @@
 }
 architecture_map.default = 'x86_64'
 
+# Older versions of yum embed M2Crypto with SSL that doesn't support TLS1.2
+prefix = node['platform_version'].to_i < 6 ? 'http' : 'https'
```

Comment:
I merely migrated the code from the repository recipe, but I just noticed that it's inside of a `case node['platform_family']` block. 

This is fixed in6eb91d7. I did it a bit differently to try to be more DRY.